### PR TITLE
[ADD] account_netting: AR/AP netting

### PR DESCRIPTION
**Pull request by pedrobaeza**
# AR/AP netting

This module allows to compensate the balance of a receivable account with the
balance of a payable account for the same partner, creating a journal item
that reflects this operation.

**WARNING**: This operation can be forbidden in your country by the accounting
regulations, so you should check current laws before using it. For example, in
Spain, this is not allowed at first instance, unless you document well the
operation from both parties.
# Usage

From any account journal entries view:
- Accounting/Journal Entries/Journal Entries
- Accounting/Periodic Processing/Reconciliation/Manual Reconciliation

select all the lines that corresponds to both AR/AP operations from the same
partner. Click on "More > Compensate". If the entries don't correspond the same
partner or they aren't AR/AP accounts, you will get an error.

On contrary, a dialog box will be presented with the result of the operation
and a selection of the journal to register the operation. When you click on the
"Compensate" button, a journal item is created with the corresponding
counterparts of the AR/AP operations

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@fclementic2c, maybe you are interested in this

**Comment by AlbertCabedo** [comment]
:+1: well done (tested on runbot)

File: account_netting/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+AR/AP netting
```
Comment (angelmoya) [inline account_netting/README.rst:6]: Put upper line according last template

File: account_netting/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (angelmoya) [inline account_netting/README.rst]: Put target

File: account_netting/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Account netting',
+    'version': '1.0',
```
Comment (angelmoya) [inline account_netting/__openerp__.py]: Version 8.0.1.0.0

**Comment by angelmoya** [comment]
Update README.md, then :+1:

**Comment by pedrobaeza** [comment]
Fixed

Can someone merge?

**Comment by angelmoya** [comment]
Good work, waitting for merge

**Comment by pedrobaeza** [comment]
I self-merge then.

**Pull request merged**